app/views/layouts/admin/layout.html.haml

```diff
@@ -2,14 +2,15 @@
 %html
   %head
     %meta{charset: "utf-8"}
-    %title TfgTemplate: Admin
+    %title= page_title
```

Comment:
The other day I found stuff in the application layout that didn't belong in there any longer and there is stuff in the admin layout that is not needed (1 example: [easy sign in](https://github.com/thefrontiergroup/rails-template/blob/feature/better-page-titles-and-active-links/app/views/layouts/admin/layout.html.haml#L54))

I am referring to these kinda things. Admin can have it's own layout, I am ok with this, but it should not be a carbon copy of the main one because not all elements are required in either layout.